dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -448,6 +449,7 @@ object LagomPlugin extends AutoPlugin with LagomPluginCompat {
     lagomServicesPortRange := defaultPortRange,
     lagomServiceLocatorEnabled := true,
     lagomServiceLocatorPort := 9008,
+    lagomServiceGatewayHost := "127.0.0.1",
     lagomServiceGatewayPort := 9000,
     lagomServiceGatewayImpl := "akka-http",
     lagomServiceLocatorUrl := s"http://localhost:${lagomServiceLocatorPort.value}",
```

Comment:
I find this set of settings a bit confusing:

```
    lagomServiceLocatorPort := 9008,
    lagomServiceGatewayHost := "127.0.0.1",
    lagomServiceGatewayPort := 9000,
    lagomServiceLocatorUrl := s"http://localhost:${lagomServiceLocatorPort.value}",
```

It's a little known fact that Service Gateway and Service Locator are a single process: the service locator is a play application which contains an object that happens to be the service gateway which listens on a different port.

The change suggested here introduces `lagomServiceGatewayHost` so users may opt out of `0.0.0.0` for the gateway binding, but I think we're missing a similar setting for Service Locator so [the `Server` creation](https://github.com/lagom/lagom/blob/62d51996cb9bbe8784a754fa571581ead701337e/dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/discovery/ServiceLocatorServer.scala#L62) can override the default too.

Reply:
So does it make sense to inject the gateway host into the locator URI à la string interpolation? This enforces that the gateway and locator are on the same host.

Reply:
OK, so add two keys -- `lagomServiceLocatorHost` and `lagomServiceGatewayHost` -- and use the new locator host key in the `lagomServiceLocatorUrl` key? Does this mean the locator URL key becomes deprecated?